packages/react-vapor/src/components/calendar/tests/Countdown.spec.tsx

```diff
@@ -15,4 +15,20 @@ describe('Countdown', () => {
         expect(container.firstChild).toHaveClass('countdown-calendar');
         expect(today).toHaveClass('todays-date');
     });
+
+    it('should display "Last day" on the last day of the month', () => {
+        const dateNow = Date.now;
+        // @ts-ignore
+        Date.now = jest.fn(() => new Date('2020-01-31T12:00:00.000Z'));
```

Comment:
Could we go with something like that instead? https://stackoverflow.com/a/63377110

We're using Jest 26 so it should be good
```suggestion
        jest
            .useFakeTimers('modern')
            .setSystemTime(new Date('2020-01-31).getTime());
```